malaria-ios/ViewControllers/GameRulesViewController.swift

```diff
@@ -0,0 +1,34 @@
+import Foundation
+import UIKit
+
+/// Presents game rules.
+class GameRulesViewController : UIViewController {
+  
+  @IBOutlet weak var gameTitleLabel: UILabel!
+  @IBOutlet weak var gameDescriptionLabel: UILabel!
+  
+  // Provided by previous viewController
+  var game: Game!
+  
+  override func prepareForSegue(segue: UIStoryboardSegue, sender: AnyObject?) {
+    let gameVC = segue.destinationViewController as! GameViewController
+    gameVC.game = game
+  }
+  
+  override func viewWillAppear(animated: Bool) {
+    super.viewWillAppear(animated)
+    
+    gameTitleLabel.text = self.game.name
+    gameDescriptionLabel.text = self.game.rules
+  }
+  
+  @IBAction func cancelBtnHandler(sender: AnyObject) {
+    view.endEditing(true)
+    dismissViewControllerAnimated(true, completion: nil)
+  }
+  
+  @IBAction func startGameBtnHandler(sender: AnyObject) {
+   performSegueWithIdentifier("Show " + game.name, sender: self)
```

Comment:
Don't forget to add in the documentation how to extend the list of games! (the segue id detail is important for that)